src/main/java/com/robinpowered/sdk/model/Event.java

```diff
@@ -275,10 +275,11 @@ public String getMimeType() {
         private final DateTime startedAt;
         private final DateTime endedAt;
         private final boolean isAllDay;
-        private final List<Invitee.Invitation> invitees;
+        private final List<Invitable> invitees;
 
         public Booking(String ownerRef, String title, String description,
-                       DateTime startedAt, DateTime endedAt, boolean isAllDay, List<Invitee.Invitation> invitees) {
+                       DateTime startedAt, DateTime endedAt, boolean isAllDay,
+                       List<Invitable> invitees) {
```

Comment:
Android studio is 100 lines

**Google's android code style rec:**
> Each line of text in your code should be at most 100 characters long.
https://source.android.com/source/code-style.html